src/Abp/AbpKernelModule.cs

```diff
@@ -212,6 +213,7 @@ private void RegisterMissingComponents()
             IocManager.RegisterIfNot<ITenantStore, NullTenantStore>(DependencyLifeStyle.Singleton);
             IocManager.RegisterIfNot<ITenantResolverCache, NullTenantResolverCache>(DependencyLifeStyle.Singleton);
             IocManager.RegisterIfNot<IEntityHistoryStore, NullEntityHistoryStore>(DependencyLifeStyle.Singleton);
+            IocManager.RegisterIfNot<IJavaScriptMinifier, NullJavaScriptMinifier>(DependencyLifeStyle.Singleton);
```

Comment:
I think we should define IJavaScriptMinifier in the Web.Common package instead of the Core Abp package.

Reply:
Yes, I will update later.